recording.js

```diff
@@ -35,16 +35,23 @@ function save(fileName, data) {
  * @param {Number} width The width of the canvas to frame
  * @param {Number} height The height of the canvas to frame
  * @param {Number} padding The border padding
+ * @param {Number} frameSize Size of title bar of the frame
+ * @param {Number} frameRadius Radius (in pixels) of the frame corners
  */
-function generateFrame(width, height, padding) {
-  padding;
-  // Create Frame
-  const baseCanvas = document.createElement('canvas');
-  baseCanvas.width = width + padding * 2;
-  baseCanvas.height = height + padding * 2;
-
-  const { backgroundColor, foregroundColor } = window.store.getState().ui;
-
+function generateFrame(width, height, padding, frameSize, frameRadius) {
+  const { backgroundColor } = window.store.getState().ui;
+
+  /* Import frame markup and hyper logo */
+  const frameMarkup = fs.readFileSync(
+    path.join(__dirname, 'frame-markup.html'),
+    'utf-8',
+  );
+  const hyperLogoSVG = fs.readFileSync(

```

Comment:
Could these two files be read in parallel?

Reply:
That’s a good call. Although these files are small, if you’d like to add parallel/async reading that would be awesome!

Reply:
I think it's fine for now. We can come back to this when it's a bigger deal.